.github/workflows/post-merge.yaml

```diff
@@ -1,20 +1,21 @@
-name: lambda-deploy
+name: post-merge
 
 on:
+  # Run on merges to master or tag pushes
   push:
     branches: [ 'master' ]
-    tags: [ 'v?[0-9]+\.[0-9]+\.[0-9]+' ]
+    tags: [ '*' ]
```

Comment:
i think this could be useful for creating test releases, and i figured that if someone with permission to create and push a tag does so, then the expectation is for a release to be created from that tag.

Reply:
compelling argument.  sounds good to me.